src/components/App/App.tsx

```diff
@@ -37,8 +37,8 @@ class App extends React.Component<{}, AllData> {
       <div className="App">
         <Header />
         <main>
-          <SortBox updateSort={this.updateSort} />
```

Comment:
Should we revert this back to show everything?

Reply:
Yes good call, fixed!